### ISSUE TITLE
Aren't nodes in the same group/community supposed to be clustered together?

### ISSUE THREAD

**Issue by aguinaldoabbj** [neo4j-contrib/neovis.js#191]
Hi, it's me again bugging you again.

From the example viz screenshot I was led to think that nodes within the same community would be clustered together in the diagram. However, after running a community detection algorithm and trying to use the group functionality, the nodes are well clustered:
![Screenshot_20210817_143740](https://user-images.githubusercontent.com/13151704/129774335-dbe271f2-d702-4ef6-9eea-65f2e5c6d054.png)
I can see some nodes with the same group property are together in several clusters spread in the diagram. I was expecting that all nodes within a group would be at least in the same region of the graph.

Is there a set of settings that would maximize this desired effect?

**Comment by thebestnom** [comment]
Not bugging at all, seeing people use the library is the reason I maintain it (not enough 😅)
Hmm... Again this looks like a vis.js thing, you mean something like https://github.com/visjs/vis-network/issues/203 ?

**Comment by aguinaldoabbj** [comment]
> Not bugging at all, seeing people use the library is the reason I maintain it (not enough )
> Hmm... Again this looks like a vis.js thing, you mean something like [visjs/vis-network#203](https://github.com/visjs/vis-network/issues/203) ?

Yeah, maybe not so well defined clusters, but something like your shot:
<img width="820" alt="example-viz" src="https://user-images.githubusercontent.com/13151704/129782624-f69bccbb-b58c-4351-9462-7933943190f0.png">

**Comment by thebestnom** [comment]
Didn't work on that 😅 neovis is a vis warper, if vis.js doesn't support that I don't think I have the capacity to support that 😅
